docs/publishing-artifacts.md

```diff
@@ -49,10 +49,9 @@ credentials += Credentials(Path.userHome / ".sbt" / "sonatype.credentials")
 
 ## Publishing a signed build
 
-Just do an `sbt publishSigned` or `sbt publishLocalSigned` from `cloudflow/core`. The `build.sbt` is set up for this. Just a few notes:
-
-* Once you do a `publishSigned` or `publishLocalSigned` you may be prompted for entering a passphrase to unlock your secret PGP code. I noticed that this prompt often gets garbled with the stdout of `sbt`. I am not sure if it's a Mac thing or it happens for some combination of versions of `sbt` and `gpg`. It happens on my machine.
-* Once the publish goes through, login to [sonatype UI](https://oss.sonatype.org) using your Sonatype credentials. You will be able to see all artifacts in a Staging Repository. Check if all contents look ok.
-* Once you have verified all contents of the staging repository, close the repository using the close button in the UI. Closing the staging repository will start a process to validate all contents and check for compliance with Maven Central. If all checks are ok, the staging repository will be closed successfully.
-* The next step is to do the release using the release button. This will release all artifacts to the Sonatype release repository. The released artifacts are now ready to be transferred to Maven Central. This may take some time and will be synced with Maven Central the next time this sync process runs.
+* In order to publish signed artifacts locally, just do `sbt publishLocalSigned`
+* In order to do a publish of signed artifacts to Sonatype and then subsequently getting it sync'd with Maven Central, just do the following 2 steps:
+ * `sbt publishSigned`, which will ask for the GPG passphrase before signing
+ * `sbt sonatypeBundleRelease`, which will do the actual publish to staging repository, promote it to final repository and subsequently making the artifacts available for Maven Central sync
```

Comment:
I'm a little bit confused by this. If we tell ourselves that these are the two steps, then what is sbt-release for? Should we always use `sbt release` to publish signed artifacts to Maven Central?

Reply:
@agemooij These 2 steps are only for the streamlet libraries to be pushed to Maven Central. But the release process involves a lot more .. in fact this is **one of the steps** in the release .. 

* pushing operator image to dockerhub
* releasing `kubectl-cloudflow` to bintray
* preparing docker images of `examples` with appropriate plugin versions and pushing them to dockerhub
* releasing helm charts
* releasing docs

Reply:
So perhaps the READMe should be a bit more clear about when to use what.

Reply:
@agemooij Added a note on this at the end of this documentation

Reply:
@agemooij The steps are exactly similar to what we do in `pipelines-release` where we use the default pipeline from `sbt-release`. We need this customization here for the Sonatype dance